### PR TITLE
Part 2

### PR DESCRIPTION
**Pull request by chancancode**

### CODE REVIEW

File: src/chapters/14-provider-components.md [hunk 1]
```diff
@@ -278,17 +282,17 @@ The `as |results|` syntax might look a little new to us, but it isn't the first
...
 
 > Zoey says...
 >
-> The local variable name `results` is arbitrary, and isn't special in any way! You could name it anything: `as |data|`, `as |filtered|`, or even `as |banana|`! Really, the important thing here is that however you name the block param is how you will have access to the yielded data from inside the block.
+> The local variable name `results` is arbitrary, and isn't special in any way! You could name it anything: `as |data|`, `as |filtered|`, or even `as |banana|`! In fact, the `as |...|` syntax is the same as declaring a local variable in JavaScript. Just as we can create a variable like `let banana = ...`, and then have access to that variable whenever we call `banana`, we can also have access to the yielded item by using whatever variable name we gave to our black parameter. The important thing here is that however you name the block param is how you will have access to the yielded data from inside the block.
```
Comment (vaidehijoshi) [inline src/chapters/14-provider-components.md]: @chancancode i have applied your suggestions in this commit; let me know if any of this still feels confusing or wordy

File: src/chapters/15-part-2-recap.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* Working with adapters and serializers
+
+<h3><a href="../14-provider-components/">Chapter 14</a></h3>
+
+* Using Ember's built-in `<Input>` helper
+* The provider component pattern
+* Using block parameters when invoking components
+* Yielding data to caller components
+
+Awesome! The concepts you learned about in part 2 of the tutorial are ones that you'll find in many production-level Ember apps. You've now taken your knowledge to the next level&mdash;and you've also finished the entire tutorial, hooray!
```
Comment (vaidehijoshi) [inline src/chapters/15-part-2-recap.md]: @chancancode I feel like there's another sentence or two that belongs here...but I'm not sure what to say. Thoughts?

Comment (vaidehijoshi) [inline src/chapters/15-part-2-recap.md]: @chancancode I feel like there's another sentence or two that belongs here...but I'm not sure what to say?

File: src/chapters/14-provider-components.md (hunk 1 above)
Comment (chancancode) [reply src/chapters/14-provider-components.md]: 1. Let's break it up into paragraphs.
2. Maybe:

> In fact, the `as |...|` syntax is the same as declaring a local variable in JavaScript. Just as we can create a variable like `let banana = ...`,

Into 

> In fact, the `... as |banana|` syntax is the same as declaring a local variable in JavaScript. Just as we can create a variable like `let banana = ...`,

File: src/chapters/15-part-2-recap.md (hunk 2 above)
Comment (chancancode) [reply src/chapters/15-part-2-recap.md]: I think ideally we will have a "next steps" section recommending additional resources. For now, maybe just recommend that the read the rest of the guides for in-depth understanding on the topics, or just go build an app of their own choosing to practice.

File: src/chapters/15-part-2-recap.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+* Ember Data models
+* Testing models
+* Loading models in routes
+* The Ember Data store
+* Working with adapters and serializers
+
+<h3><a href="../14-provider-components/">Chapter 14</a></h3>
+
+* Using Ember's built-in `<Input>` helper
```
Comment (chancancode) [inline src/chapters/15-part-2-recap.md]:
```suggestion
* Using Ember's built-in `<Input>` component
```

Comment (vaidehijoshi) [reply src/chapters/15-part-2-recap.md]: looks like we had this in the provide component chapter as well. i just pushed up a fix for both instances

**Comment by chancancode** [comment]
Thanks @vaidehijoshi @gitKrystan @eramod!!

**Pull request merged**